app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -1104,6 +1103,16 @@ fun MessageList(
                         isSelectedMessage = (message.header.messageId == selectedMessageId),
                         isInteractionAvailable = interactionAvailability == InteractionAvailability.ENABLED
                     )
+
+                    if ((index == 0 && lazyPagingMessages.itemCount == 1) || (index + 1) == lazyPagingMessages.itemCount) {
```

Comment:
Just to make it a bit more readable:

```suggestion
                    val isTheOnlyItem = index == 0 && lazyPagingMessages.itemCount == 1
                    val isTheLastItem = (index + 1) == lazyPagingMessages.itemCount
                    if (isTheOnlyItem || isTheLastItem) {
```